netpyne/metadata/metadata.py

```diff
@@ -643,10 +643,10 @@
                         "children": {
                             "pop": {
                                 "label": "Target Population",
-                                "help": "Select the population targets.",
+                                "help": "Select the population targets e.g. {'pop': ['Exc1', 'Exc2']}",
                                 "suggestions": "",
                                 "hintText": "",
-                                "type": "list(float)"
+                                "type": "str"
```

Comment:
Can you refresh my mind? Why did you decide to change this? (Sorry I think you mentioned sth regarding a bug but I can't remember)

Reply:
no, sorry my bad.